packages/flutter_tools/test/base/build_test.dart

```diff
@@ -406,7 +406,6 @@ void main() {
         '--url_mapping=dart:vmservice_io,${fs.path.join(skyEnginePath, 'sdk_ext', 'vmservice_io.dart')}',
         '--embedder_entry_points_manifest=$kVmEntrypoints',
         '--embedder_entry_points_manifest=$kIoEntries',
-        '--dependencies=${fs.path.join(outputPath, 'snapshot.d')}',
```

Comment:
What's the reason for this change?  Is it because that's the default?

Reply:
gen_snapshot now fails when asked to generate a depfile in Dart 2 mode since it can't do it.